amostra/server/engine.py

```diff
@@ -338,13 +338,13 @@ def post(self):
             uids.append(data['uid'])
             res = database.container.insert(data)
             if res:
-                database.request.create_index([('uid', pymongo.DESCENDING)],
-                                                 unique=True, background=True)
-                database.request.create_index([('time', pymongo.DESCENDING)],
-                                            unique=False, background=True)
+                database.container.create_index([('uid', pymongo.DESCENDING)],
```

Comment:
Does this sensible short circuit?  It just seems a waste to add two extra network hits on _every_ insert.